docs/manifest.json

```diff
@@ -1,212 +1,254 @@
 [
```

Comment:
Can we add a comment at the top of this file explaining that it was auto-generated? I bet someone will update it manually soon 😃

Reply:
Unfortunately, comment are not allowed in JSON?

Reply:
😿